actionlist/app/actions/types.js

```diff
@@ -1,6 +1,8 @@
-export const TOGGLE_DRAWER = 'toggle_drawer';
+export const OPEN_DRAWER = 'open_drawer';
+export const CLOSE_DRAWER = 'close_drawer';
 export const FILTER_ACTION_LIST = 'filter_action_list';
 export const SORT_ACTION_LIST = 'sort_action_list';
 export const SELECT_DROPDOWN_OPTION = 'select_dropdown_option';
 export const RESET_FILTERS = 'reset_filters';
+export const DATE_FILTER_CHANGE = 'date_filter_change';
```

Comment:
No, there's no true namespacing with action types, it's really just naming them with path-style naming so you're less likely to have collisions and so it's obvious from reading the code what they do and how they're related to other actions. So it would just look like
```es6
export const FILTER_ACTION_LIST = 'action_list/filter/apply';
export const FILTER_DATE = 'action_list/filter/date_change';
export const FILTER_RESET = 'action_list/filter/reset';
export const PREFERENCES_DRAWER_OPEN = 'preferences/drawer/open';
export const PREFERENCES_DRAWER_CLOSE = 'preferences/drawer/close';
export const PREFERENCES_RESET = 'preferences/reset';
// etc
```